openvino_tensorflow/tf_conversion_extensions/src/conversion_extensions.hpp

```diff
@@ -22,10 +22,10 @@ OP_CONVERTER(translate_depthwise_conv_2d_native_op);
 OP_CONVERTER(translate_mkl_swish_op);
 OP_CONVERTER(translate_concat_op);
 OP_CONVERTER(translate_non_max_suppression_op);
-OP_CONVERTER(translate_slice_op);
 OP_CONVERTER(translate_cast_op);
 OP_CONVERTER(translate_ctc_greedy_decoder_op);
 OP_CONVERTER(translate_sparse_to_dense_op);
+OP_CONVERTER(translate_parallel_dynamic_stitch_op);
```

Comment:
Is the issue with ParallelDynamicSwitch solved now on OV master? If so, should we consider removing this extension?